scripts/inner/kubestellar

```diff
@@ -63,10 +73,10 @@ process_running() {
 }
 
 wait_for_process(){
-  status=$(process_running $1)
-  MAX_RETRIES=10
-  retries=0
-  status_code=0
+  local status=$(process_running $1)
+  local MAX_RETRIES=10
```

Comment:
General comment: we should standardize on variable names - upper vs lower case.

Reply:
There is already a lot of mixed usage, I am not sure it is worth the trouble. Does the fact that the usage is mixed cause a problem?

Reply:
Just looks unprofessional.  But you are right and we have bigger fish to fry.

Reply:
Let's push it for a (much) later PR :-)